cmake/configs/modes.cmake

```diff
@@ -123,10 +123,15 @@ if(MSVC)
     set(asan_link_flags "/fsanitize=address")
     set(asan_compiler_c_flag "/fsanitize=address")
     set(asan_compiler_cxx_flag "/fsanitize=address")
+    set(asan_lib_flags "")
 else()
-    set(asan_link_flags "-fsanitize=address")
     set(asan_compiler_c_flag "-fsanitize=address")
     set(asan_compiler_cxx_flag "-fsanitize=address")
+    set(asan_link_flags "-fsanitize=address")
+    if(GNU_C_COMPILER AND GNU_CXX_COMPILER)
+        set(asan_link_flags "${asan_link_flags}")
```

Comment:
Why is this line necessary?

Reply:
Oh whoops, good catch. That must've been left over from debugging. Removed in 544b10d.